pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -214456,8 +214456,8 @@ self: {
      }:
      mkDerivation {
        pname = "nixfmt";
-       version = "0.5.0";
-       sha256 = "0rxi8zrd2xr72w673nvgnhb0g3r7rssc1ahlhz8rmdpc6c1a82wl";
+       version = "0.6.0";
+       sha256 = "XwJ2A1Hzg4pznlWhxm1heCvinJgfBC5ZCTcaLC5D70k=";
```

Comment:
@SebTM Thanks for looking into this.

This file is actually automatically-generated, so any manual changes will get blown away.

`nixfmt` has already been updated to 0.6.0 in https://github.com/NixOS/nixpkgs/pull/266650, so it will be available in `master` when that PR is merged in.